src/WebJobs.Script/Binding/WebJobsCoreScriptBindingProvider.cs

```diff
@@ -1,298 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.ObjectModel;
-using System.IO;
-using System.Net.Http;
-using Microsoft.Azure.WebJobs.Host;
-using Microsoft.Azure.WebJobs.Script.Binding;
-using Microsoft.Azure.WebJobs.Script.Binding.Http;
-using Microsoft.Azure.WebJobs.Script.Extensibility;
-using Newtonsoft.Json.Linq;
-
-namespace Microsoft.Azure.WebJobs.Script
-{
-    /// <summary>
-    /// Enables all Core SDK Triggers/Binders
-    /// </summary>
-    [CLSCompliant(false)]
-    public class WebJobsCoreScriptBindingProvider : ScriptBindingProvider
```

Comment:
Most of this completely disappears and becomes rule-based. We do have a new ExtensionBase class in the SDK that handles some of this glue.